api/server/router/system/system_routes.go

```diff
@@ -20,6 +20,21 @@ import (
 	"golang.org/x/net/context"
 )
 
+var (
```

Comment:
I wonder if this validation should be there, or `SubscribeToEvents` instead. Thinking about that because most of the filter validation for other endpoint that supports filtering are down lower (in the backend, a.k.a. daemon) — see `daemon/list.go` for example.

At least we should be consistent to where we put those 👼